styles/Editor.css

```diff
@@ -942,13 +942,16 @@
   vertical-align: super;
   font-size: 75%;
 }
+.ProseMirror .deleted.pending.footnote::before, .ProseMirror .deleted.accepted.footnote::before {
```

Comment:
Can you please explain this?

Reply:
for adding strike-through style for inline footnotes 
![image](https://github.com/Atypon-OpenSource/manuscripts-body-editor/assets/93189633/3ef17363-06d0-4030-b6e7-d90f3d5fc7e0)

Reply:
Ok, to achieve this please use existing styles. Remove this `.ProseMirror .deleted.pending.footnote::before, .ProseMirror .deleted.accepted.footnote::before` from here and check `AdvancedEditor.css` you will see that we already have a general:
```
.ProseMirror .deleted.pending {
  background: var(--pending-bg-color);
  color: var(--deleted-color);
  text-decoration: line-through;
}
.ProseMirror .deleted.accepted {
  background: var(--accepted-bg-color);
  color: var(--deleted-color);
  text-decoration: line-through;
}
```

Just update these and make it:

```
.ProseMirror .deleted.pending::before,
.ProseMirror .deleted.pending {
  background: var(--pending-bg-color);
  color: var(--deleted-color);
  text-decoration: line-through;
}

.ProseMirror .deleted.accepted::before,
.ProseMirror .deleted.accepted {
  background: var(--accepted-bg-color);
  color: var(--deleted-color);
  text-decoration: line-through;
}
```

Reply:
done